.gitignore

```diff
@@ -11,3 +11,7 @@ target/*
 .hg/*
 test-output/*
 
+.idea
```

Comment:
IMO it's not a good practice to add IDE specific rules. I even wonder wheter this .gitignore file should be committed in the repository.